.changeset/good-pugs-greet.md

```diff
@@ -0,0 +1,5 @@
+---
+"@cultureamp/rich-text-toolkit": major
+---
+
+Update prosemirror dependencies and type declarations. Adds prosemirror packages as named exports from the toolkit
```

Comment:
In the future we should probably include a bit of info on why we're doing a major version release here, and how to work through any breaking changes. But it might be difficult with this one and we don't have any direct consumers of this so I'm happy to leave this.

Reply:
Yeah that's a fair point, at the very least I will update the PR descriptions to at least have a record here of why this was done 👍